### ISSUE TITLE
groupfolders break users list

### ISSUE THREAD

**Issue by wdfee** [nextcloud/groupfolders#241]
With group folders app activated the users list doesn't show any user (only numbers at groups).
error notice: "An error occured during the request. Unable to proceed."
error in nextcloud log:
"TypeError: Argument 5 passed to OCA\GroupFolders\Mount\MountProvider::getMount() must implement interface OCP\Files\Cache\ICacheEntry, null given, called in /var/www/vhosts/[mydomain]/httpdocs/apps/groupfolders/lib/Mount/MountProvider.php on line 63 at /var/www/vhosts/[mydomain]/httpdocs/apps/groupfolders/lib/Mount/MountProvider.php#67"

NC Version: 14.0.4.2 (fresh install)
PHP version: 7.2.10
Apache / Ubuntu 18.04.1
Group folders: 2.0.0

**Comment by ElpyDE** [comment]
Same here.

NC Version: 14.0.4.2 (not a fresh install 😄)
PHP version: 7.0.33
Apache 2.4.37 / Ubuntu 14.04.5
Group folders: 2.0.0

Problem first occured after update of Group folders app. I changed other things, too. But this seems the most likely cause. Especially, deactivating the Group folders app makes the user list work again. Re-activating the app again produces the error.

**Comment by ElpyDE** [comment]
Seems like the changes in 3dff78783e34730120cd89a7e1bc504c934e1f4b introduced the error.

As __hotfix__ for the following change worked:

In groupfolders/lib/Folder/FolderManager.php change line 186 from

```php
'rootCacheEntry' => (isset($folder['fileid'])) ? Cache::cacheEntryFromData($folder, $this->mimeTypeLoader): null
```

to

```php
'rootCacheEntry' => Cache::cacheEntryFromData($folder, $this->mimeTypeLoader)
```

There are certainly other implications that I don't see. For example `occ files:scan --all` still produces a Fatal Error. But the web interface seems to work - for the moment.

Hopefully, @icewind1991 can to take a look at this 😉

**Comment by SebSeifert** [comment]
> In groupfolders/lib/Folder/FolderManager.php change line 186 from
> 
> 'rootCacheEntry' => (isset($folder['fileid'])) ? Cache::cacheEntryFromData($folder, $this->mimeTypeLoader): null
> to
> 
> 'rootCacheEntry' => Cache::cacheEntryFromData($folder, $this->mimeTypeLoader)

Yes the Interface is Loading but the Group Folder ist not shown and not created in "nc-datafolder/__groupfolders/"

**Comment by icewind1991** [comment]
This should be fixed with https://github.com/nextcloud/groupfolders/commit/fd6e598ba334928ea2b10c0a1a23495a4b0a0dfc

**Comment by icewind1991** [comment]
Released 2.0.1 which should fix this issue, if you're still experiencing this issue with that version feel free to reopen

**Comment by ElpyDE** [comment]
Thanks very much for the fast fix and release!
